src/index.ts

```diff
@@ -1,12 +1,26 @@
 import express = require('express');
 import bodyParser = require('body-parser');
 import { performLogin } from './telalogin/login';
+import { getList } from './onlineusers/allusers';
+import { Pool } from 'pg';
 
 const app: express.Application = express();
 app.use(bodyParser.json())
 
+export let pool = new Pool({
```

Comment:
Esses valores não deveriam ser trocados, certo? Se for o caso, melhor trocar pra `const` então.